apps/agent-tars/src/renderer/src/agent/EventManager.ts

```diff
@@ -50,7 +50,7 @@ export class EventManager {
     willNotifyUpdate = true,
   ): Promise<EventItem> {
     const event: EventItem = {
-      id: uuidv4(),
+      id: nanoid(),
```

Comment:
Are there any other reasons for replacement besides the smaller package size?

Reply:
High speed of generating and enough security and randomness